docs/troubleshooting/troubleshooting.mdx

```diff
@@ -62,6 +64,17 @@ Another common issue during sign-in is not receiving a verification code when us
    - Use this code in Sidestore to complete the sign-in process.
 
 
+## Can't Refresh or Install Apps
+
+#### Resolving No Wi-Fi or VPN Error
+
+1. Disable your cellular data / connect to a stable Wi-Fi network.
+2. Make sure Wi-Fi is not restricted (such as school Wi-Fi).
```

Comment:
maybe something closer to:

"Ensure that Apple domains, and the domain for your anisette server, are not restricted (e.g. school/work Wi-Fi)"